### PR TITLE
Fixed Issue #20234

### PR DESCRIPTION
**Pull request by ghost**
### Description (*)

Review text Should be capitalized

### Fixed Issues (if relevant)

1. magento/magento2#<20234>: Review text Should be capitalized

### Manual testing scenarios (*)
1. Navigate to product listing page.
2. Change the mode from grid to list.
3. Now you can see "add your review"  and "review" text in this format 
   Add Your Review
   Review

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/magento2?pullRequest=22694) <br/>All committers have signed the CLA.

**Comment by Karlasa** [comment]
@magento-engcom-team give me 2.3-develop instance

**Comment by magento-engcom-team** [comment]
Hi @Karlasa. Thank you for your request. I'm working on Magento 2.3-develop instance for you

**Comment by magento-engcom-team** [comment]
Hi @Karlasa, here is your [Magento instance](https://i-22694-2-3-develop.instances.magento-community.engineering).
Admin access: [https://i-22694-2-3-develop.instances.magento-community.engineering/admin](https://i-22694-2-3-develop.instances.magento-community.engineering/admin)
Login: `admin` Password: `123123q`
Instance will be terminated in up to 3 hours.

**Comment by Karlasa** [comment]
Hi!
@surbhi-ranosys could you please sign CLA, so wo could continue processing this pull request :) 

Cheers, 
Karla

File: app/design/frontend/Magento/luma/Magento_Catalog/web/css/source/module/_listings.less [hunk 1]
```diff
@@ -123,7 +123,7 @@
                 .reviews-actions {
                     font-size: @font-size__s;
                     margin-top: 5px;
-                    text-transform: lowercase;
+                    text-transform: none;
```
Comment (Karlasa) [inline app/design/frontend/Magento/luma/Magento_Catalog/web/css/source/module/_listings.less]: How about just removing text-transform? :)

**Comment by ghost** [comment]
Hello @Karlasa 

I have signed CLA, Can you please check now?

**Comment by Karlasa** [comment]
@surbhi-ranosys, Thank You CLA is signed, could you now look the comment I left in code review :)

File: app/design/frontend/Magento/luma/Magento_Catalog/web/css/source/module/_listings.less (hunk 1 above)
Comment (ghost) [reply app/design/frontend/Magento/luma/Magento_Catalog/web/css/source/module/_listings.less]: @Karlasa  Yes we can remove text-transform too

Comment (Karlasa) [reply app/design/frontend/Magento/luma/Magento_Catalog/web/css/source/module/_listings.less]: @surbhi-ranosys can you do that then? ;)

Comment (ghost) [reply app/design/frontend/Magento/luma/Magento_Catalog/web/css/source/module/_listings.less]: @Karlasa Yes sure

Comment (ghost) [reply app/design/frontend/Magento/luma/Magento_Catalog/web/css/source/module/_listings.less]: @Karlasa I have removed text transform can you please check

**Review by Karlasa — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @Karlasa, thank you for the review.
[ENGCOM-5120](https://jira.corp.magento.com/browse/ENGCOM-5120) has been created to process this Pull Request

**Comment by soleksii** [comment]
Hi @surbhi-ranosys !
Could you please correct text transform and on **Blank** theme too!?
blank

**Thanks!**

**Comment by ghost** [comment]
Hi @stoleksiy 

I have corrected the text transform for blank them too. Can you please verify it now?

**Comment by soleksii** [comment]
> Hi @stoleksiy
> 
> I have corrected the text transform for blank them too. Can you please verify it now?

Ok. Thanks!

**Comment by soleksii** [comment]
:heavy_check_mark: QA Passed

**Pull request merged**